middleware/jwt_test.go

```diff
@@ -526,3 +529,68 @@ func TestJwtInvalidKeys(t *testing.T) {
 	tests.Assert(t, err == nil)
 	tests.Assert(t, strings.Contains(s, "signature is invalid"))
 }
+
+func TestJwtWrongSigningMethod(t *testing.T) {
+	// Setup jwt
+	c := &JwtAuthConfig{}
+	c.Admin.PrivateKey = "Key"
+	c.User.PrivateKey = "UserKey"
+	j := NewJwtAuth(c)
+	tests.Assert(t, j != nil, "NewJwtAuth failed")
+
+	// Setup middleware framework
+	n := negroni.New(j)
+	tests.Assert(t, n != nil, "negroni.New failed")
+
+	// Create a simple middleware to check if it was called
+	called := false
+	mw := func(rw http.ResponseWriter, r *http.Request) {
+		data := context.Get(r, "jwt")
+		tests.Assert(t, data != nil, "context.Get failed")
+
+		token := data.(*jwt.Token)
+		claims := token.Claims.(jwt.MapClaims)
+		tests.Assert(t, claims["iss"] == "admin",
+			`expected claims["iss"] == "admin", got:`, claims["iss"])
+
+		called = true
+
+		rw.WriteHeader(http.StatusOK)
+	}
+	n.UseHandlerFunc(mw)
+
+	// Create test server
+	ts := httptest.NewServer(n)
+
+	// Create token
+	token := jwt.NewWithClaims(jwt.SigningMethodPS256, jwt.MapClaims{
+		// Set issuer
+		"iss": "admin",
+
+		// Set issued at time
+		"iat": time.Now().Unix(),
+
+		// Set expiration
+		"exp": time.Now().Add(time.Second * 10).Unix(),
+	})
+
+	// Should we use a fake source of randomness instead of real
+	// rand.Reader here?
+	pk, err := rsa.GenerateKey(rand.Reader, 256*2)
+	tests.Assert(t, err == nil, "rsa.GenerateKey failed:", err)
+	tokenString, err := token.SignedString(pk)
+	tests.Assert(t, err == nil, "token.SignedString failed:", err)
+
+	// Setup header
+	req, err := http.NewRequest("GET", ts.URL, nil)
+	tests.Assert(t, err == nil, "http.NewRequest failed:", err)
+
+	// Miss 'bearer' string
```

Comment:
I think this is a copy'n'paste of a copy'n'paste error, right? We *do* add the bearer string...

I.e. if my understanding is correct,  this comment should be removed. :-)